resources/measures/hvac_nighttime_oa_controls/measure.rb

```diff
@@ -0,0 +1,206 @@
+# insert your copyright here
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/
+
+# start the measure
+class NighttimeOAControls < OpenStudio::Measure::ModelMeasure
+  # human readable name
+  def name
+    # Measure name should be the title case of the class name.
+    return 'Nighttime OA Controls'
+  end
+
+  # human readable description
+  def description
+    return 'This measure sets minimum outdoor airflow to zero during extended periods of no occupancy (nighttime). Fans cycle during these unoccupied periods to meet the thermostat setpoints.'
+  end
+
+  # human readable description of modeling approach
+  def modeler_description
+    return 'This measure continues to allow for air-side economizing during nighttime hours, since it is only the minimum outdoor air level being modified.'
+  end
+
+  # define the arguments that the user will input
+  def arguments(model)
+    args = OpenStudio::Measure::OSArgumentVector.new
+
+    # the name of the space to add to the model
+    # space_name = OpenStudio::Measure::OSArgument.makeStringArgument('space_name', true)
+    # space_name.setDisplayName('New space name')
+    # space_name.setDescription('This name will be used as the name of the new space.')
+    # args << space_name
```

Comment:
delete section of unused code.

Reply:
Done--sorry, I think some of the changes I'd made in the meantime didn't get in here.